recipe/meta.yaml

```diff
@@ -10,43 +10,60 @@ source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha256: {{ sha256 }}
+  patches:
+    - 0001-remove-obsolete-components.patch
 
 build:
   number: 0
   script: {{ PYTHON }} -m pip install . --no-deps -vv
-  noarch: python
```

Comment:
Skip `py<36` https://github.com/plotly/dash/blob/680a9b05a14ecaac15cf09d3f8838d3750ed7915/setup.py#L29 as we use `contextvars ==2.4  # [py<37]` in `run`

Reply:
Good catch, thank you!